.github/workflows/label.yml

```
@@ -0,0 +1,57 @@
+name: "Pull Request Labeler"
+on:
+  pull_request_target:
+    types: [opened, reopened]
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Label based on changed files
+      uses: actions/labeler@v3
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+        # https://github.com/actions/labeler/issues/112#issuecomment-1000491676
+        sync-labels: ''
+
+    # Label based on branch name
+    - uses: actions-ecosystem/action-add-labels@v1
+      if: startsWith(github.event.pull_request.head.ref, 'doc/') || startsWith(github.event.pull_request.head.ref, 'docs')
+      with:
+        labels: documentation
```

I think these label names should match the ones in the repo and are case sensitive. For example, pymapdl has:

```suggestion
        labels: Documentation
```

Previous suggestion applies for the rest of the labels too.